features/step_definitions/navigation.rb

```diff
@@ -0,0 +1,23 @@
+Then (/^I logout/) do
+  if current_activity() == "CommCareHomeActivity"
+    touch("android.support.v7.widget.CardView index:4")
```

Comment:
The 'contains text' approach seems easier to understand, but I've been running into strange issues so I wanted to try this out.